### PR TITLE
Add Lockpicking

### PR DESCRIPTION
**Pull request by meitar**
**https://github.com/meitar/awesome-lockpicking**

**A curated list of awesome guides, tools, and other resources related to the security and compromise of locks, safes, and keys.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- [x] The entry in the Awesome list should:
	- [ ] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- [x] The list I'm submitting complies with these requirements:
	- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- [x] It's the result of hard work and the best I could possibly produce.
	- [x] Non-generated Markdown file in a GitHub repo.
	- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- [x] Not a duplicate.
	- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
	- [ ] Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- [x] Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] Has a Table of Contents section.
		- [x] Should be named `Contents`, not `Table of Contents`.
		- [x] Should be the first section in the list.
		- [x] Should only have one level of sub-lists, preferably none.
	- [x] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- [x] That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [x] If you use a license badge, it should be SVG, not PNG.
	- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- [x] The file should be named `contributing.md`. Casing is up to you.
	- [x] Has consistent formatting and proper spelling/grammar.
		- [x] Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Include a short description about the project/theme of the list. **It should not describe the list itself.**

This is not done.

---

Include a link to the Wikipedia article https://en.wikipedia.org/wiki/Lock_picking

---

Would be nice to include some videos: https://www.google.no/search?q=lockpicking&oq=lockpicking&aqs=chrome..69i57j0j69i61j0l2j69i60.1454j0j1&sourceid=chrome&ie=UTF-8#q=lockpicking&tbm=vid

**Comment by meitar** [comment]
> > Include a short description about the project/theme of the list. It should not describe the list itself.
> 
> This is not done.

Is the following description not exactly this? "A curated list of awesome guides, tools, and other resources related to the security and compromise of locks, safes, and keys." If not, what is meant by "Include a short description about the project/theme of the list" in the contributing instructions? I have read them numerous times so unless I am misinterpreting the intent, I thought this *was* done.

Either way, [I have added a brief description of lock picking itself following the quote in the awesome list's own README](https://github.com/meitar/awesome-lockpicking/commit/1e3353cd46464b2c0381b8f0a9e17e9f5905687b).

> Include a link to the Wikipedia article https://en.wikipedia.org/wiki/Lock_picking

Done, see above.

> Would be nice to include some videos.

Agreed, but I haven't found an awesome video yet, and we already link to a collection of good videos by Deviant Ollam in the how-to section. Maybe in the future someone will contribute links to specific, awesome, one-off videos on YouTube or elsewhere. :)

**Comment by sindresorhus** [comment]
> Is the following description not exactly this?

Read the whole thing in context. It should be a description in this Awesome list, not in yours.

> but I haven't found an awesome video yet

Here are the most popular ones on YouTube: https://www.youtube.com/results?q=lock+picking&sp=CANQFA%253D%253D Could probably find a few good ones there.

**Comment by meitar** [comment]
> It should be a description in this Awesome list, not in yours.

Ah, I see. Commit 65463a8d07e29893c4f7fa961aa9c228b1658119 adds the list's description to *this* list's item description.

File: readme.md [hunk 1]
```diff
@@ -406,7 +406,7 @@
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
-- [Lockpicking](https://github.com/meitar/awesome-lockpicking)
+- [Lockpicking](https://github.com/meitar/awesome-lockpicking) - Resources relating to the security and compromise of locks, safes, and keys.
```
Comment (sindresorhus) [inline readme.md]: The description should be about "lockpicking", not the list itself.

Comment (meitar) [inline readme.md]: @sindresorhus I've changed this to "Techniques and tools for opening a lock without the key and without damaging or destroying the lock or its door." and pushed an updated commit.

**Comment by sindresorhus** [comment]
The description is a bit long. What do you think about:

> The art of unlocking a lock by manipulating its components without the key.

**Comment by meitar** [comment]
I like that. :) Pushed a new commit with it.

**Pull request merged**